modules/oc-mirror-setting-incremental-mirroring.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="setting-incremental-mirroring_{context}"]
+= Setting up for incremental mirroring
+
+You mirror images to install, upgrade, and maintain {product-title} in a disconnected environment. Because the volume of the image set needed to install or upgrade {product-title} can be substantial, oc-mirror keeps metadata about previously mirrored images in its storage backend thus reducing the volume of the image sets to the recent images only.
```

Comment:
```suggestion
You mirror images to install, upgrade, and maintain {product-title} in a disconnected environment. Because the volume of the image set needed to install or upgrade {product-title} can be substantial, oc-mirror stores the metadata of the previously mirrored images in its storage backend thus reducing the volume of the image sets to the recent images only.
```